### ISSUE TITLE
部署了XX-Net 3.2.8的gae服务端后，下载文件只能下载32m，或者错误

### ISSUE THREAD

**Issue by jujf1111** [XX-net/XX-Net#4531]
部署了XX-Net 3.2.8的gae服务端后，下载文件只能下载32m，或者错误
Nov 10 21:19:18.865 - [INFO] PAC from:127.0.0.1:5607 GET /proxy.pac
Nov 10 21:19:18.898 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:19.336 - [DEBUG] 59.18.35.224 fail:timeout('timed out',)
Nov 10 21:19:19.336 - [DEBUG] report_connect_fail:59.18.35.224
Nov 10 21:19:19.336 - [DEBUG] get ip:59.18.35.111 t:221
Nov 10 21:19:19.873 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:21.872 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:22.534 - [DEBUG] create_ssl update ip:59.18.35.111 time:127 h2:0
Nov 10 21:19:22.626 - [DEBUG] ip:210.2.185.97 real fail
Nov 10 21:19:23.491 - [DEBUG] 59.18.35.171 worker close:keep alive
Nov 10 21:19:23.872 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:24.637 - [WARNING] server app return fail, status:502
Nov 10 21:19:25.642 - [DEBUG] GAE GET https://codeload.github.com/Microsoft/Windows-driver-samples/zip/master
Nov 10 21:19:25.872 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:27.627 - [DEBUG] ip:59.18.35.220 real fail
Nov 10 21:19:27.871 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:28.386 - [DEBUG] GAE_Exception 605 'status:500'
Nov 10 21:19:28.386 - [WARNING] gae_exception:GAE_Exception() https://codeload.github.com/Microsoft/Windows-driver-samples/zip/master
Nov 10 21:19:28.872 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:29.866 - [INFO] PAC from:127.0.0.1:5607 GET /proxy.pac
Nov 10 21:19:30.873 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:32.627 - [DEBUG] ip:115.254.121.209 real fail
Nov 10 21:19:32.873 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:34.035 - [DEBUG] restore ip:203.82.77.206
Nov 10 21:19:34.891 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:36.896 - [DEBUG] GAEProxy Web_control 127.0.0.1:5515 GET /deploy?cmd=get_log
Nov 10 21:19:37.724 - [DEBUG] 106.103.1.144 keep alive fail, inactive_time:76 head_timeout:21
Nov 10 21:19:37.733 - [DEBUG] report_connect_fail:106.103.1.144
Nov 10 21:19:38.890 - [DEBUG] GAEProxy Web_control 127.0.0.1:5516 GET /deploy?cmd=get_log
Nov 10 21:19:39.181 - [DEBUG] ip:61.219.131.238 real fail

**Comment by jzp820927** [comment]
可能是因为 GAE 的限制造成的。这个还真不好解决。

**Comment by jujf1111** [comment]
你的意思是谷歌造成的

**Comment by jzp820927** [comment]
是的。你可以看看这个  https://github.com/XX-net/XX-Net/wiki/GoAgent-Limitation

**Comment by twofisher** [comment]
我前段时间用chrome从github下载zip也碰到这个问题，后来用firefox下载成功了，但没有再次确认是否与浏览器有关。

**Comment by jzp820927** [comment]
小问题下载，应该没啥问题。

**Comment by twofisher** [comment]
@jzp820927 大的，54M

**Comment by jzp820927** [comment]
那可能就是对某些特定网站的兼容性有问题。
